#! /bin/sh -e
## 7_devscripts-debuild-uc-us.dpatch by Roland Mas <lolando@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Invoke debuild with appropriate options to not sign the package

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
diff -urNad 7.emacs-goodies-el.tmp/elisp/devscripts-devscripts.el 7.emacs-goodies-el/elisp/devscripts.el
--- 7.emacs-goodies-el.tmp/elisp/devscripts-el/devscripts.el	2003-03-19 22:09:47.000000000 +0100
+++ 7.emacs-goodies-el/elisp/devscripts-el/devscripts.el	2003-03-19 22:26:48.000000000 +0100
@@ -9,7 +9,7 @@
     (switch-to-buffer debuild-buffer)
     (kill-region (point-min) (point-max))
     (compilation-mode)
-    (start-process debuild-process debuild-buffer "/usr/bin/debuild")))
+    (start-process debuild-process debuild-buffer "/usr/bin/debuild" "-uc" "-us")))
 
 (defun debi ()
   "Run debi in the current directory, to install debian packages generated by previous invocation of debuild"
